.github/workflows/buildOnlyModified.yml

```diff
@@ -18,6 +18,14 @@ jobs:
       - name: extract branch
         id: extract_branch
         run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      - name: Patch Docker Daemon data-root
```

Comment:
I will add a comment with this link : 
https://github.community/t/worker-running-out-of-disk-space/18245

Just to understand, why we do that and why this code

Reply:
the docker patch has been removed so this is not required anymore